tests/tools.py

```diff
@@ -1,15 +1,15 @@
 import subprocess
 
 
-def run(cmd, error=False):
+def run(cmd, error=False, stderr=subprocess.PIPE):
```

Comment:
all these changes, just because i wrote a test that was intended to drop  out/errormessages?

then please have if changed like this: 

```patch
- def run(cmd, error=False, stderr=subprocess.PIPE):
8 def run(cmd, error=False, *, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
```

this way each stream can be dopped/piped individually,
and the `*` causes the downstream users to be forced to name the param, like 
```
run('echo foo', out=subprocess.PIPE)



-----



but all of this was not needed, if this function could simply return stdout and stderr separately, as a tuple.

```py
def run(...)>
  return out, err

_, err = run('echo 1 && foobar')
out, _ = run('echo 1') 
out, err = run('echo 1 && foobar')
```

Reply:
stdout is now also a parameter